modules/creating-a-project-using-the-CLI.adoc

```diff
@@ -13,10 +13,7 @@ If allowed by your cluster administrator, you can create a new project.
 Projects starting with `openshift-` and `kube-` are considered critical by {product-title}. As such, {product-title} does not allow you to create Projects starting with `openshift-` or `kube-` using the `oc new-project` command. Cluster administrators can create these Projects using the `oc adm new-project` command.
 ====
 
-[NOTE]
-====
-You cannot assign an SCC to pods created in one of the default namespaces: `default`, `kube-system`, `kube-public`, `openshift-node`, `openshift-infra`, and `openshift`. You cannot use these namespaces for running pods or services.
-====
+include::snippets/default-namespaces.adoc[]
```

Comment:
Yeah, I just replaced all of them since they had the warnings there already. But I agree that this does seem like overkill. Since this is the only assembly that uses these modules, I think we're safe to have the warning just at the top of the assembly. Removing these duplicates